### DISCUSSION TITLE
全局热键如何设置音量大小？

### DISCUSSION THREAD

**Discussion by cc19264997** [shuax/MouseInc.Settings#285] [General]
全局热键如何设置音量大小？ 不习惯滚轮

**Comment by shuax** [comment]
你多研究一下吧，参考下已有手势

**Comment by cc19264997** [comment]
感谢回复，已经解决&nbsp; &nbsp;大爱MouseInc




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "shuax/MouseInc.Settings"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月9日(星期四) 上午6:15
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shuax/MouseInc.Settings] 全局热键如何设置音量大小？ (Discussion #285)





 
你多研究一下吧，参考下已有手势
